packages/e2e-tests/scripts/save-examples.ts

```diff
@@ -153,21 +153,24 @@ async function saveExamples(
       continue;
     }
 
-    const [_, runtime] = buildId.split("-");
+    const [_, buildRuntime] = buildId.split("-");
 
     let category: TestExampleFile["category"];
     let folder: TestExampleFile["folder"];
+    let runtime: TestExampleFile["runtime"];
 
-    switch (runtime) {
+    switch (buildRuntime) {
       case "chromium":
       case "gecko": {
```

Comment:
FWIW this was intentional, but we should probably throw instead if "gecko" is passed in as a target.